python/lsst/pipe/tasks/calibrate.py

```diff
@@ -121,6 +131,13 @@ class CalibrateConnections(pipeBase.PipelineTaskConnections, dimensions=("instru
         dimensions=("instrument", "visit", "detector"),
     )
 
+    outputSummaryMetrics = cT.Output(
+        doc="Summary metrics created by the calibration task",
+        name="calexpSummary_metrics",
+        storageClass="MetricMeasurementBundle",
+        dimensions=("instrument", "visit", "detector"),
+    )
+
```

Comment:
I would write out the SummaryStats too if someone wants to create the metrics later. I don't know why we already don't, maybe @erykoff knows. Is it just because we ARE writing it out with the exposure?

Reply:
I'm actually confused about this whole thing.  You can get the summary stats as a component of a calexp with `calexp.summaryStats` in a connection without reading the full calexp.  I'm not sure why this code is in `calibrate.py` at all (beyond the circular import problems).  Furthermore, we are moving from `calibrate.py` to `calibrateImage.py` (which is already the default in AP) but I don't think this belongs there either.

Reply:
It's so that the SummaryStats are written to metrics during the calibrate task, which I understand is something that both @ctslater and @mfisherlevine would like. I agree that grabbing the stats is easy and quick post-calibrate, but it still involves read overhead, which is avoided by having it in-task like this. AFAIA, it's always been a feature of analysis_tools that they can be used in-task like this.

The reason it's in calibrate.py rather than calibrateImage.py is simply because this is where I started developing it. It can be easily added to calibrateImage.py, and I think @ctslater is planning to make a ticket to do that.